controllers/managedocs_controller.go

```diff
@@ -1048,6 +1059,16 @@ func (r *ManagedOCSReconciler) reconcilePrometheus() error {
 			},
 			Key: alertRelabelConfigSecretKey,
 		}
+		r.prometheus.Spec.ExternalLabels = map[string]string{"app": "managed-ocs"}
```

Comment:
```suggestion
		r.prometheus.Spec.ExternalLabels = map[string]string{monLabelKey: monLabelValue}
```
I think we should use constants to assign externalLabels.